src/main/webapp/charts.js

```diff
@@ -0,0 +1,46 @@
+google.charts.load('current', {'packages':['corechart']});
```

Comment:
this file is already in the other pr right?

Reply:
shudhatma asked me to split the PR so i moved this here in new PR